rapids-metadata.json

```diff
@@ -2965,6 +2958,392 @@
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            },
+            "cugraph-service-client": {
+              "has_conda_package": true,
+              "has_cuda_suffix": false,
+              "has_wheel_package": false,
+              "publishes_prereleases": true
+            },
+            "cugraph-service-server": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": false,
+              "publishes_prereleases": true
+            },
+            "libcugraph": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            },
+            "libcugraph_etl": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": false,
+              "publishes_prereleases": true
+            },
+            "pylibcugraph": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            }
+          }
+        },
+        "cugraph-docs": {
+          "packages": {}
+        },
+        "cugraph-gnn": {
+          "packages": {
+            "cugraph-pyg": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            },
+            "libwholegraph": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            },
+            "pylibwholegraph": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            }
+          }
+        },
+        "cuml": {
+          "packages": {
+            "cuml": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            },
+            "libcuml": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
+              "publishes_prereleases": true
+            },
+            "libcuml-tests": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": false,
+              "publishes_prereleases": true
+            }
+          }
+        },
+        "cumlprims_mg": {
+          "packages": {
+            "libcumlprims": {
+              "has_conda_package": true,
+              "has_cuda_suffix": true,
+              "has_wheel_package": true,
```

Comment:
We don't have wheels for cumlprims_mg, just conda packages. I'm pretty sure it's shipped as part of cuml wheels.